backend/src/crud.py

```diff
@@ -72,7 +72,7 @@ def read_listings(db: Session, skip: int = 0, limit: int = 100) -> list[Listing]
 def update_listing(
     db: Session, listing_id: int, new_listing_data: ListingRead
 ) -> Listing:
-    db_listing = read_user(db, listing_id)
+    db_listing = read_listing(db, listing_id)
```

Comment:
Because the PUT allows for owner_id, we probably need to do verification of user as well.

Reply:
@jameswinegar good call, I added the user check to put listing(): https://github.com/winlinworks/fairbnb/pull/30/commits/e8f9e7e4aae57667b432de02cccc092ef6ba4a6c